File: back/app/models/cliente.py
from app.extensions import db
from app.models.base import BaseModel



class Cliente(BaseModel):
  __tablename__ = 'clientes'

  nombre = db.Column(db.String(120), nullable=False)
  telefono = db.Column(db.String(40))
  email = db.Column(db.String(120))
  estado = db.Column(db.String(40), default='activo')

  def __repr__(self):
    return f'<Cliente {self.nombre}>'

File: desktop/src/PipitaDesktop/Models/Cliente.cs
namespace PipitaDesktop.Models;

public sealed class Cliente : BaseEntity
{
    public string Nombre { get; set; } = string.Empty;
    public string? Telefono { get; set; }
    public string? Email { get; set; }
    public string Estado { get; set; } = "activo";

    public ICollection<Vehiculo> Vehiculos { get; set; } = new List<Vehiculo>();
    public ICollection<SolicitudCliente> Solicitudes { get; set; } = new List<SolicitudCliente>();
    public ICollection<TrabajoDistribuidora> TrabajosDistribuidora { get; set; } = new List<TrabajoDistribuidora>();
    public ICollection<Cita> Citas { get; set; } = new List<Cita>();
}


